README.md

```diff
@@ -1,2 +1,274 @@
+
+Run the following commands to wait until the workflow builds are done and workflows are running on namespace sonataflow-infra:
+  oc wait -n sonataflow-infra sonataflow/event-timeout --for=condition=Built --timeout=15m
+  oc wait -n sonataflow-infra sonataflow/event-timeout --for=condition=Running --timeout=5m
+```
+Wait and watch for the deployed components to be up and running. You will notice that a pod `event-timeout-1-build` will start running. SonataFlow operator uses this pod to build and push the images for the event-timout workflow into the local registry. Be patient, as this can take 10 to 20 minutes. Once it is done another pod for the workflow gets started. 
+```console
+$ oc wait -n openshift-serverless deploy/knative-openshift --for=condition=Available --timeout=5m
+  oc wait -n knative-eventing knativeeventing/knative-eventing --for=condition=Ready --timeout=5m
+  oc wait -n knative-serving knativeserving/knative-serving --for=condition=Ready --timeout=5m
+  oc wait -n openshift-operators deploy/sonataflow-operator-controller-manager --for=condition=Available --timeout=5m
+  oc wait -n sonataflow-infra sonataflowplatform/sonataflow-platform --for=condition=Succeed --timeout=5m
+  oc wait -n sonataflow-infra deploy/data-index --for=condition=Available --timeout=5m
+  oc wait -n sonataflow-infra deploy/jobs-service --for=condition=Available --timeout=5m
+  oc wait -n sonataflow-infra pod/postgres-db-0 --for=condition=Ready --timeout=5m
+  oc wait -n orchestrator-install pod/orchestrator-postgresql-0 --for=condition=Ready --timeout=5m
+  oc wait -n orchestrator-install deploy/orchestrator-backstage --for=condition=Available --timeout=5m
+
+deployment.apps/knative-openshift condition met
+knativeeventing.operator.knative.dev/knative-eventing condition met
+knativeserving.operator.knative.dev/knative-serving condition met
+deployment.apps/sonataflow-operator-controller-manager condition met
+sonataflowplatform.sonataflow.org/sonataflow-platform condition met
+deployment.apps/data-index condition met
+deployment.apps/jobs-service condition met
+pod/postgres-db-0 condition met
+pod/orchestrator-postgresql-0 condition met
+deployment.apps/orchestrator-backstage condition met
+```
+Run the following commands to wait until the workflow builds are done and workflows are running:
+```console
+$ oc wait -n sonataflow-infra sonataflow/event-timeout --for=condition=Built --timeout=15m
+  oc wait -n sonataflow-infra sonataflow/event-timeout --for=condition=Running --timeout=5m
+
+sonataflow.sonataflow.org/event-timeout condition met
+sonataflow.sonataflow.org/event-timeout condition met
+```
+## Testing the Sample Workflow - Event Timeout
+
+* Retrieve the route of the event timeout workflow service
+```shell
+$ oc get route event-timeout -n sonataflow-infra -o=jsonpath='{.spec.host}'
+```
+Sample output:
+```
+event-timeout-sonataflow-infra.apps.cluster-ffmlv.dynamic.opentlc.com
```

Comment:
To avoid copy and paste, is it possible to create an env var to use in the following commands? (Or any other approach that can ease a bit :) )